orttraining/orttraining/python/training/ortmodule/_custom_op_symbolic_registry.py

```diff
@@ -12,6 +14,86 @@
 from onnxruntime.training import ortmodule
 
 
+def _enforce_check(condition, message):
```

Comment:
Is this code movement required? Can we keep this file only for custom symbolic registration, and move all helper functions to another utility/helper file?

Reply:
Yeah, let me clean up the code in it and avoid adding more things than the fix only needs.

My original intention is, PythonOp is one of the operators to export, so maybe we should move some of the shared custom export logic into _custom_op_symbolic_registry.py. While looks we treat this file as a workaround collection against exporter defect. If so, I would move those changes unrelated to the fix to its original places, and just leave the fix here. If possible, those fix should also be up streamed to torch exporter later.  FYI @iK1D